### ISSUE TITLE
Hi, I have some questions regarding this project

### ISSUE THREAD

**Issue by Joe23232** [kazcw/cn-stratum#1]
Hi, I have some questions regarding this project:

1. Am I able to import it as a library/crate into my own Rust code and integrate it into my own software so I can compile all into a single executable?
2. Does it support either [Beam](https://beam.mw/),  [Zcash](https://z.cash/) or [Monera](https://www.getmonero.org/) or any other privacy respecting cryptocurrency?
3. How efficient is it compared to something like [gminer](https://github.com/develsoftware/GMinerRelease)?
4. Do you guys get a commission from my cryptomining if I were to use this library/crate?
5. How well maintained is this project? How many people are working on it?

**Comment by kazcw** [comment]
1. Yes, it's dual-licensed MIT/Apache2 as noted in the Cargo.toml. I haven't got around to a LICENSE file.
2. If they use the same stratum protocol as monero
3. This is a stratum protocol client library
4. No, the library just does what you tell it to
5. It's just me. I wrote it as part of a miner I made for an algorithm that's obsolete now. The stratum protocol seldom changes so it probably still works, but I haven't had any reason to maintain it recently.
